StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/WalletHeaderView.swift

```diff
@@ -63,11 +63,12 @@ extension PaymentSheetViewController {
 
         private lazy var payWithLinkButton: PayWithLinkButton = {
             let button = PayWithLinkButton()
-            if LiquidGlassDetector.isEnabled {
+            if appearance.cornerRadius == nil, LiquidGlassDetector.isEnabledInMerchantApp {
                 button.ios26_applyCapsuleCornerConfiguration()
             } else {
-                button.cornerRadius = appearance.cornerRadius
+                button.cornerRadius = appearance.cornerRadius ?? PaymentSheet.Appearance.defaultCornerRadius
             }
+            button.applyCornerRadiusOrConfiguration(for: appearance, ios26DefaultCornerStyle: .capsule)
             button.accessibilityIdentifier = "pay_with_link_button"
```

Comment:
Seems like we can get rid of this if/else, and just call the line you have on line 71:
```
button.applyCornerRadiusOrConfiguration(for: appearance, ios26DefaultCornerStyle: .capsule)
```

Reply:
Sadly PayWithLinkButton has a special `cornerRadius` property whereas the helper (and ~all other users) set `layer.cornerRadius`.

But good catch, I don't think we should be setting line 71 at all.